crates/sui-graphql-rpc/src/types/address.rs

```diff
@@ -164,10 +165,11 @@ impl Address {
         };
 
         TransactionBlock::paginate(
-            ctx.data_unchecked(),
+            ctx,
             page,
             filter,
             self.checkpoint_viewed_at,
+            Some(within_checkpoints.unwrap_or(10000000)),
```

Comment:
Noting that the max scan limit should be 100_000_000, and that it's fine have it be `None`, so we probably shouldn't have logic anywhere to set it if it hasn't been provided.